README.md

```diff
@@ -103,7 +103,7 @@ OMI, please see repository [Build-omi](https://github.com/Microsoft/Build-omi).
 
 Installing OMI configures a daemon named "omid" which can be
 controlled with standard service controllers: `service` or
-`systemctl`. Additionally, a service_control script can be found at:
+`systemctl`. Additionally, a `service_control script can be found at:
```

Comment:
incorrect format, lost end char ` for service_contronl!

Reply:
Crud, missed that. I'll fix that now.